pkg/cidata/cloud-config.yaml

```diff
@@ -0,0 +1 @@
+cidata.TEMPLATE.d/user-data
```

Comment:
Do we still need this symlink?
You can just mv user-data cloud-config.yaml

Reply:
No, it was not possible to embed a symlink